generate_sl.py

```diff
@@ -71,21 +74,20 @@ def main():
     parser.add_argument('-ll', '--station_file',
                         help="Path to a single station file for ruputure distance calculations")
     parser.add_argument('-np', '--processes', default=DEFAULT_N_PROCESSES, help="number of processors to use")
-    parser.add_argument('-ml', '--max_line', default=33, type=int, help="maximum number of lines in a slurm script")
-    parser.add_argument('rrup_output_dir', help="directory containing rupture distances output")
-
+    parser.add_argument('-ml', '--max_lines', default=100, type=int, help="maximum number of lines in a slurm script. Default 100")
+    parser.add_argument('-e', '--extended', action='store_const', const='-e', default='', help="add '-e' to indicate the use of extended pSA period. Default not using")
+    parser.add_argument('-simple', action='store_const',const='-s',default='',
```

Comment:
should be double '--' for long options.
Either way should follow convention from calculate_ims.py